x-pack/test/observability_ai_assistant_api_integration/tests/complete/complete.spec.ts

```diff
@@ -299,6 +310,79 @@ export default function ApiTest({ getService }: FtrProviderContext) {
       });
     });
 
+    describe('after executing a screen context action', async () => {
+      let events: StreamingChatResponseEvent[];
+
+      before(async () => {
+        events = await getEvents(
+          {
+            screenContexts: [
+              {
+                actions: [
+                  {
+                    name: 'my_action',
+                    description: 'My action',
+                    parameters: {
+                      type: 'object',
+                      properties: {
+                        foo: {
+                          type: 'string',
+                        },
+                      },
+                    },
+                  },
+                ],
+              },
+            ],
+          },
+          async (conversationSimulator) => {
+            await conversationSimulator.next({
+              function_call: { name: 'my_action', arguments: JSON.stringify({ foo: 'bar' }) },
+            });
+            await conversationSimulator.complete();
+          }
+        );
+      });
+
+      it('closes the stream without persisting the conversation', () => {
+        expect(
+          pick(
+            events[events.length - 1],
+            'message.message.content',
+            'message.message.function_call',
+            'message.message.role'
+          )
+        ).to.eql({
+          message: {
+            message: {
+              content: '',
+              function_call: {
+                name: 'my_action',
+                arguments: JSON.stringify({ foo: 'bar' }),
+                trigger: MessageRole.Assistant,
+              },
+              role: MessageRole.Assistant,
+            },
+          },
+        });
+      });
+
+      it('does not store the conversation', async () => {
+        expect(
+          events.filter((event) => event.type === StreamingChatResponseEventType.ConversationCreate)
+            .length
+        ).to.eql(0);
+
+        const conversations = await observabilityAIAssistantAPIClient
+          .writeUser({
+            endpoint: 'POST /internal/observability_ai_assistant/conversations',
```

Comment:
Why is this POST rather than GET?

Reply:
it's a search request, with optionally a body with filters (right now just a query string).